numcodecs/bitround.py

```diff
@@ -39,8 +39,7 @@ class BitRound(Codec):
     codec_id = 'bitround'
 
     def __init__(self, keepbits: int):
-        if keepbits < 0:
-            raise ValueError("keepbits must be zero or positive")
+        assert keepbits >= 0, "keepbits must be zero or positive"
```

Comment:
This is a different concrete exception which has the (slight) potential of changing semantics

Reply:
@martindurant, could you please submit a PR with a test then (as I believe this module was something you added)? The test coverage dropped on this line

Reply:
Ah, so that was the rationale for changing it...

https://github.com/zarr-developers/numcodecs/pull/393

Reply:
```suggestion
        if keepbits < 0:
            raise ValueError("keepbits must be zero or positive")
```